controllers/topics.controller.js

```diff
@@ -0,0 +1,12 @@
+const { selectTopics } = require("../models/topics.model");
+
+exports.getTopics = (req, res, next) => {
+	selectTopics()
+        .then((topics) => {
+			res.send({ topics });
+		})
+		.catch((err) => {
+			console.log(err);
+			next(err);
+		});
+};
```

Comment:
Good work here too. Same feedback as above: good function name, and sending the response back on a key of `topics` is a good choice.

Great to see you've implemented a catch block. You don't need the `console.log(err)` in here, just invoke `next()` and move on. Let the error handling middleware console log any errors (if you need or want it to).

Reply:
This console log was something I picked up when working on the sprint Thursday and Friday and is purely for dev purposes. I'll add a comment above it to help me not forgot to remove it.